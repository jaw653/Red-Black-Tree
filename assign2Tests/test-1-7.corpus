hfcq
sxtx
elwn
ydqs
rmdi
nhaj
lvpg
gczn
kbfc
bycf
latj
dlex
zjfn
rhye
eolk
smac
qfhr
gjwr
jrdp
fjme
sttj
bsqh
ibsa
pudf
bmwh
xvbi
ogxt
plah
ftti
njpv
mjwb
dbhf
nfok
arsp
xsko
dkyk
erst
ckpo
tnrz
oyed
eupg
mjxl
bich
uara
rkut
wjjp
waom
burh
pgnd
plor
vqyp
srsj
bmez
xppv
qfjt
caar
gnwv
amov
fonx
gfjj
tpjs
fanx
fyqj
yqae
dybg
lsbs
gqpm
xyyq
qhkv
hzso
xkyv
aabf
zflm
xpef
fvsd
wqvm
zihi
hbwh
lucn
xgsy
ngkm
vqrc
llhj
dfvc
nekw
fjds
fifc
qacd
impd
eihs
upbx
wxcl
dohj
xldd
vjhl
lkot
yfyf
nfxj
wajs
zndd
bnob
argv
angn
zwga
defs
meci
endg
aijb
xxcx
oisr
yagx
ynzb
teuf
iyqp
lvvl
femd
drat
bvmz
vtzu
gyxb
frjn
rbec
wznd
fcgi
vjcw
eoyb
jzxp
zxte
qcui
dykb
aaef
enqb
wuxc
kxfu
wfjw
cecs
gwal
xnmx
ptet
guuc
otgz
rltp
qens
jqlr
ondn
apmp
irkq
ngsc
aydt
mwkc
cyxn
qkfg
ziua
zirk
zean
kvrm
vufh
sqmw
qlki
xpow
zkya
tqku
wnjg
icvd
wanq
sbnk
mzsj
qhfp
tero
wcls
ruaz
xvev
ytos
uddg
cxrs
gxja
daob
ebtv
wvwv
tdst
yilt
lqbp
otju
qsuv
vlwb
mrzk
mxhh
acaz
kmuw
hfcq
sxtx
elwn
ydqs
rmdi
nhaj
lvpg
gczn
kbfc
bycf
latj
dlex
zjfn
rhye
eolk
smac
qfhr
gjwr
jrdp
fjme
sttj
bsqh
ibsa
pudf
bmwh
xvbi
ogxt
plah
ftti
njpv
mjwb
dbhf
nfok
arsp
xsko
dkyk
erst
ckpo
tnrz
oyed
eupg
mjxl
bich
uara
rkut
wjjp
waom
burh
pgnd
plor
vqyp
srsj
bmez
xppv
qfjt
caar
gnwv
amov
fonx
gfjj
tpjs
fanx
fyqj
yqae
dybg
lsbs
gqpm
xyyq
qhkv
hzso
xkyv
aabf
zflm
xpef
fvsd
wqvm
zihi
hbwh
lucn
xgsy
ngkm
vqrc
llhj
dfvc
nekw
fjds
fifc
qacd
impd
eihs
upbx
wxcl
dohj
xldd
vjhl
lkot
yfyf
nfxj
wajs
zndd
bnob
argv
angn
zwga
defs
meci
endg
aijb
xxcx
oisr
yagx
ynzb
teuf
iyqp
lvvl
femd
drat
bvmz
vtzu
gyxb
frjn
rbec
wznd
fcgi
vjcw
eoyb
jzxp
zxte
qcui
dykb
aaef
enqb
wuxc
kxfu
wfjw
cecs
gwal
xnmx
ptet
guuc
otgz
rltp
qens
jqlr
ondn
apmp
irkq
ngsc
aydt
mwkc
cyxn
qkfg
ziua
zirk
zean
kvrm
vufh
sqmw
qlki
xpow
zkya
tqku
wnjg
icvd
wanq
sbnk
mzsj
qhfp
tero
wcls
ruaz
xvev
ytos
uddg
cxrs
gxja
daob
ebtv
wvwv
tdst
yilt
lqbp
otju
qsuv
vlwb
mrzk
mxhh
acaz
kmuw
