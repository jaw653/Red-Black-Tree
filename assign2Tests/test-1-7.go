f hfcq
f sxtx
f elwn
f ydqs
f rmdi
f nhaj
f lvpg
f gczn
f kbfc
f bycf
f latj
f dlex
f zjfn
f rhye
f eolk
f smac
f qfhr
f gjwr
f jrdp
f fjme
f sttj
f bsqh
f ibsa
f pudf
f bmwh
f xvbi
f ogxt
f plah
f ftti
f njpv
f mjwb
f dbhf
f nfok
f arsp
f xsko
f dkyk
f erst
f ckpo
f tnrz
f oyed
f eupg
f mjxl
f bich
f uara
f rkut
f wjjp
f waom
f burh
f pgnd
f plor
f vqyp
f srsj
f bmez
f xppv
f qfjt
f caar
f gnwv
f amov
f fonx
f gfjj
f tpjs
f fanx
f fyqj
f yqae
f dybg
f lsbs
f gqpm
f xyyq
f qhkv
f hzso
f xkyv
f aabf
f zflm
f xpef
f fvsd
f wqvm
f zihi
f hbwh
f lucn
f xgsy
f ngkm
f vqrc
f llhj
f dfvc
f nekw
f fjds
f fifc
f qacd
f impd
f eihs
f upbx
f wxcl
f dohj
f xldd
f vjhl
f lkot
f yfyf
f nfxj
f wajs
f zndd
f bnob
f argv
f angn
f zwga
f defs
f meci
f endg
f aijb
f xxcx
f oisr
f yagx
f ynzb
f teuf
f iyqp
f lvvl
f femd
f drat
f bvmz
f vtzu
f gyxb
f frjn
f rbec
f wznd
f fcgi
f vjcw
f eoyb
f jzxp
f zxte
f qcui
f dykb
f aaef
f enqb
f wuxc
f kxfu
f wfjw
f cecs
f gwal
f xnmx
f ptet
f guuc
f otgz
f rltp
f qens
f jqlr
f ondn
f apmp
f irkq
f ngsc
f aydt
f mwkc
f cyxn
f qkfg
f ziua
f zirk
f zean
f kvrm
f vufh
f sqmw
f qlki
f xpow
f zkya
f tqku
f wnjg
f icvd
f wanq
f sbnk
f mzsj
f qhfp
f tero
f wcls
f ruaz
f xvev
f ytos
f uddg
f cxrs
f gxja
f daob
f ebtv
f wvwv
f tdst
f yilt
f lqbp
f otju
f qsuv
f vlwb
f mrzk
f mxhh
f acaz
f kmuw
s
r
