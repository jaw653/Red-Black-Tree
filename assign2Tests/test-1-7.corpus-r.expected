Frequency of hfcq: 2
Frequency of sxtx: 2
Frequency of elwn: 2
Frequency of ydqs: 2
Frequency of rmdi: 2
Frequency of nhaj: 2
Frequency of lvpg: 2
Frequency of gczn: 2
Frequency of kbfc: 2
Frequency of bycf: 2
Frequency of latj: 2
Frequency of dlex: 2
Frequency of zjfn: 2
Frequency of rhye: 2
Frequency of eolk: 2
Frequency of smac: 2
Frequency of qfhr: 2
Frequency of gjwr: 2
Frequency of jrdp: 2
Frequency of fjme: 2
Frequency of sttj: 2
Frequency of bsqh: 2
Frequency of ibsa: 2
Frequency of pudf: 2
Frequency of bmwh: 2
Frequency of xvbi: 2
Frequency of ogxt: 2
Frequency of plah: 2
Frequency of ftti: 2
Frequency of njpv: 2
Frequency of mjwb: 2
Frequency of dbhf: 2
Frequency of nfok: 2
Frequency of arsp: 2
Frequency of xsko: 2
Frequency of dkyk: 2
Frequency of erst: 2
Frequency of ckpo: 2
Frequency of tnrz: 2
Frequency of oyed: 2
Frequency of eupg: 2
Frequency of mjxl: 2
Frequency of bich: 2
Frequency of uara: 2
Frequency of rkut: 2
Frequency of wjjp: 2
Frequency of waom: 2
Frequency of burh: 2
Frequency of pgnd: 2
Frequency of plor: 2
Frequency of vqyp: 2
Frequency of srsj: 2
Frequency of bmez: 2
Frequency of xppv: 2
Frequency of qfjt: 2
Frequency of caar: 2
Frequency of gnwv: 2
Frequency of amov: 2
Frequency of fonx: 2
Frequency of gfjj: 2
Frequency of tpjs: 2
Frequency of fanx: 2
Frequency of fyqj: 2
Frequency of yqae: 2
Frequency of dybg: 2
Frequency of lsbs: 2
Frequency of gqpm: 2
Frequency of xyyq: 2
Frequency of qhkv: 2
Frequency of hzso: 2
Frequency of xkyv: 2
Frequency of aabf: 2
Frequency of zflm: 2
Frequency of xpef: 2
Frequency of fvsd: 2
Frequency of wqvm: 2
Frequency of zihi: 2
Frequency of hbwh: 2
Frequency of lucn: 2
Frequency of xgsy: 2
Frequency of ngkm: 2
Frequency of vqrc: 2
Frequency of llhj: 2
Frequency of dfvc: 2
Frequency of nekw: 2
Frequency of fjds: 2
Frequency of fifc: 2
Frequency of qacd: 2
Frequency of impd: 2
Frequency of eihs: 2
Frequency of upbx: 2
Frequency of wxcl: 2
Frequency of dohj: 2
Frequency of xldd: 2
Frequency of vjhl: 2
Frequency of lkot: 2
Frequency of yfyf: 2
Frequency of nfxj: 2
Frequency of wajs: 2
Frequency of zndd: 2
Frequency of bnob: 2
Frequency of argv: 2
Frequency of angn: 2
Frequency of zwga: 2
Frequency of defs: 2
Frequency of meci: 2
Frequency of endg: 2
Frequency of aijb: 2
Frequency of xxcx: 2
Frequency of oisr: 2
Frequency of yagx: 2
Frequency of ynzb: 2
Frequency of teuf: 2
Frequency of iyqp: 2
Frequency of lvvl: 2
Frequency of femd: 2
Frequency of drat: 2
Frequency of bvmz: 2
Frequency of vtzu: 2
Frequency of gyxb: 2
Frequency of frjn: 2
Frequency of rbec: 2
Frequency of wznd: 2
Frequency of fcgi: 2
Frequency of vjcw: 2
Frequency of eoyb: 2
Frequency of jzxp: 2
Frequency of zxte: 2
Frequency of qcui: 2
Frequency of dykb: 2
Frequency of aaef: 2
Frequency of enqb: 2
Frequency of wuxc: 2
Frequency of kxfu: 2
Frequency of wfjw: 2
Frequency of cecs: 2
Frequency of gwal: 2
Frequency of xnmx: 2
Frequency of ptet: 2
Frequency of guuc: 2
Frequency of otgz: 2
Frequency of rltp: 2
Frequency of qens: 2
Frequency of jqlr: 2
Frequency of ondn: 2
Frequency of apmp: 2
Frequency of irkq: 2
Frequency of ngsc: 2
Frequency of aydt: 2
Frequency of mwkc: 2
Frequency of cyxn: 2
Frequency of qkfg: 2
Frequency of ziua: 2
Frequency of zirk: 2
Frequency of zean: 2
Frequency of kvrm: 2
Frequency of vufh: 2
Frequency of sqmw: 2
Frequency of qlki: 2
Frequency of xpow: 2
Frequency of zkya: 2
Frequency of tqku: 2
Frequency of wnjg: 2
Frequency of icvd: 2
Frequency of wanq: 2
Frequency of sbnk: 2
Frequency of mzsj: 2
Frequency of qhfp: 2
Frequency of tero: 2
Frequency of wcls: 2
Frequency of ruaz: 2
Frequency of xvev: 2
Frequency of ytos: 2
Frequency of uddg: 2
Frequency of cxrs: 2
Frequency of gxja: 2
Frequency of daob: 2
Frequency of ebtv: 2
Frequency of wvwv: 2
Frequency of tdst: 2
Frequency of yilt: 2
Frequency of lqbp: 2
Frequency of otju: 2
Frequency of qsuv: 2
Frequency of vlwb: 2
Frequency of mrzk: 2
Frequency of mxhh: 2
Frequency of acaz: 2
Frequency of kmuw: 2
0: nhaj-2-B(nhaj-2-B)-
1: elwn-2-B(nhaj-2-B)-l rmdi-2-B(nhaj-2-B)-r
2: bycf-2-B(elwn-2-B)-l hfcq-2-R(elwn-2-B)-r plah-2-B(rmdi-2-B)-l wqvm-2-R(rmdi-2-B)-r
3: bich-2-R(bycf-2-B)-l dkyk-2-R(bycf-2-B)-r fjme-2-B(hfcq-2-R)-l latj-2-B(hfcq-2-R)-r ondn-2-B(plah-2-B)-l qfhr-2-R(plah-2-B)-r uara-2-B(wqvm-2-R)-l xsko-2-B(wqvm-2-R)-r
4: amov-2-B(bich-2-R)-l bmwh-2-B(bich-2-R)-r cyxn-2-B(dkyk-2-R)-l dybg-2-B(dkyk-2-R)-r fanx-2-R(fjme-2-B)-l gczn-2-R(fjme-2-B)-r iyqp-2-R(latj-2-B)-l mjwb-2-R(latj-2-B)-r ogxt-2-R(ondn-2-B)-l oyed-2-R(ondn-2-B)-r pudf-2-B(qfhr-2-R)-l qlki-2-B(qfhr-2-R)-r sxtx-2-R(uara-2-B)-l vtzu-2-R(uara-2-B)-r xkyv-2-B(xsko-2-B)-l zflm-2-R(xsko-2-B)-r
5: aaef-2-R(amov-2-B)-l argv-2-R(amov-2-B)-r =bmez-2-B(bmwh-2-B)-l bsqh-2-R(bmwh-2-B)-r ckpo-2-R(cyxn-2-B)-l defs-2-R(cyxn-2-B)-r dohj-2-B(dybg-2-B)-l ebtv-2-B(dybg-2-B)-r erst-2-B(fanx-2-R)-l fifc-2-B(fanx-2-R)-r ftti-2-B(gczn-2-R)-l gqpm-2-B(gczn-2-R)-r ibsa-2-B(iyqp-2-R)-l jrdp-2-B(iyqp-2-R)-r lucn-2-B(mjwb-2-R)-l nfok-2-B(mjwb-2-R)-r =njpv-2-B(ogxt-2-R)-l =oisr-2-B(ogxt-2-R)-r otgz-2-B(oyed-2-R)-l =pgnd-2-B(oyed-2-R)-r plor-2-B(pudf-2-B)-l qcui-2-B(pudf-2-B)-r qhkv-2-R(qlki-2-B)-l rhye-2-R(qlki-2-B)-r srsj-2-B(sxtx-2-R)-l tnrz-2-B(sxtx-2-R)-r vqrc-2-B(vtzu-2-R)-l waom-2-B(vtzu-2-R)-r wznd-2-R(xkyv-2-B)-l xpef-2-R(xkyv-2-B)-r ydqs-2-B(zflm-2-R)-l zjfn-2-B(zflm-2-R)-r
6: =aabf-2-B(aaef-2-R)-l aijb-2-B(aaef-2-R)-r angn-2-B(argv-2-R)-l arsp-2-B(argv-2-R)-r =bnob-2-B(bsqh-2-R)-l burh-2-B(bsqh-2-R)-r caar-2-B(ckpo-2-R)-l =cxrs-2-B(ckpo-2-R)-r dbhf-2-B(defs-2-R)-l =dfvc-2-B(defs-2-R)-r =dlex-2-R(dohj-2-B)-l =drat-2-R(dohj-2-B)-r =dykb-2-R(ebtv-2-B)-l =eihs-2-R(ebtv-2-B)-r eolk-2-R(erst-2-B)-l =eupg-2-B(erst-2-B)-r femd-2-B(fifc-2-B)-l =fjds-2-B(fifc-2-B)-r fonx-2-B(ftti-2-B)-l fyqj-2-B(ftti-2-B)-r gjwr-2-R(gqpm-2-B)-l gyxb-2-R(gqpm-2-B)-r =hzso-2-B(ibsa-2-B)-l impd-2-B(ibsa-2-B)-r =jqlr-2-B(jrdp-2-B)-l kbfc-2-R(jrdp-2-B)-r llhj-2-R(lucn-2-B)-l lvvl-2-B(lucn-2-B)-r mwkc-2-R(nfok-2-B)-l ngkm-2-B(nfok-2-B)-r =otju-2-R(otgz-2-B)-r =ptet-2-R(plor-2-B)-r =qacd-2-R(qcui-2-B)-l =qens-2-R(qcui-2-B)-r qfjt-2-B(qhkv-2-R)-l =qkfg-2-B(qhkv-2-R)-r rbec-2-B(rhye-2-R)-l rkut-2-B(rhye-2-R)-r smac-2-R(srsj-2-B)-l =sttj-2-B(srsj-2-B)-r tero-2-B(tnrz-2-B)-l tpjs-2-B(tnrz-2-B)-r vjcw-2-R(vqrc-2-B)-l =vqyp-2-B(vqrc-2-B)-r wajs-2-B(waom-2-B)-l wjjp-2-R(waom-2-B)-r wvwv-2-B(wznd-2-R)-l =xgsy-2-B(wznd-2-R)-r xldd-2-B(xpef-2-R)-l xppv-2-B(xpef-2-R)-r xxcx-2-R(ydqs-2-B)-l ynzb-2-R(ydqs-2-B)-r zirk-2-B(zjfn-2-B)-l zwga-2-R(zjfn-2-B)-r
7: =acaz-2-R(aijb-2-B)-l =apmp-2-R(angn-2-B)-r =aydt-2-R(arsp-2-B)-r =bvmz-2-R(burh-2-B)-r =cecs-2-R(caar-2-B)-r =daob-2-R(dbhf-2-B)-l endg-2-B(eolk-2-R)-l =eoyb-2-B(eolk-2-R)-r =fcgi-2-R(femd-2-B)-l =frjn-2-R(fonx-2-B)-r =fvsd-2-R(fyqj-2-B)-l =gfjj-2-B(gjwr-2-R)-l =gnwv-2-B(gjwr-2-R)-r gwal-2-B(gyxb-2-R)-l =hbwh-2-B(gyxb-2-R)-r =icvd-2-R(impd-2-B)-l =irkq-2-R(impd-2-B)-r =jzxp-2-B(kbfc-2-R)-l kvrm-2-B(kbfc-2-R)-r =lkot-2-B(llhj-2-R)-l lsbs-2-B(llhj-2-R)-r =lvpg-2-R(lvvl-2-B)-l =meci-2-R(lvvl-2-B)-r mjxl-2-B(mwkc-2-R)-l mzsj-2-B(mwkc-2-R)-r =nfxj-2-R(ngkm-2-B)-l =ngsc-2-R(ngkm-2-B)-r =qhfp-2-R(qfjt-2-B)-r =qsuv-2-R(rbec-2-B)-l =rltp-2-R(rkut-2-B)-r sbnk-2-B(smac-2-R)-l =sqmw-2-B(smac-2-R)-r =tdst-2-R(tero-2-B)-l =teuf-2-R(tero-2-B)-r =tqku-2-R(tpjs-2-B)-r upbx-2-B(vjcw-2-R)-l vjhl-2-B(vjcw-2-R)-r =vufh-2-R(wajs-2-B)-l =wanq-2-R(wajs-2-B)-r wfjw-2-B(wjjp-2-R)-l =wnjg-2-B(wjjp-2-R)-r =wuxc-2-R(wvwv-2-B)-l =wxcl-2-R(wvwv-2-B)-r =xnmx-2-R(xldd-2-B)-r =xpow-2-R(xppv-2-B)-l xvbi-2-B(xxcx-2-R)-l xyyq-2-B(xxcx-2-R)-r yfyf-2-B(ynzb-2-R)-l ytos-2-B(ynzb-2-R)-r =zihi-2-R(zirk-2-B)-l =ziua-2-R(zirk-2-B)-r zndd-2-B(zwga-2-R)-l =zxte-2-B(zwga-2-R)-r
8: =enqb-2-R(endg-2-B)-r =guuc-2-R(gwal-2-B)-l =gxja-2-R(gwal-2-B)-r =kmuw-2-R(kvrm-2-B)-l =kxfu-2-R(kvrm-2-B)-r =lqbp-2-R(lsbs-2-B)-l =mrzk-2-R(mjxl-2-B)-r =mxhh-2-R(mzsj-2-B)-l =nekw-2-R(mzsj-2-B)-r =ruaz-2-R(sbnk-2-B)-l =uddg-2-R(upbx-2-B)-l =vlwb-2-R(vjhl-2-B)-r =wcls-2-R(wfjw-2-B)-l =xvev-2-R(xvbi-2-B)-r =yagx-2-R(xyyq-2-B)-r =yilt-2-R(yfyf-2-B)-r =yqae-2-R(ytos-2-B)-l =zean-2-R(ytos-2-B)-r =zkya-2-R(zndd-2-B)-l
Words/Phrases: 378
Nodes: 189
Minimum depth: 6
Maximum depth: 9
